docs/sources/installation/helm/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Install Grafana Loki with Helm
+menuTitle: Concept
+description: Use this template when you write a concept topic.
+aliases:
+  - /docs/writers-toolkit/latest/templates/concept-template
+weight: 100
+keywords:
+  - helm 
+  - scalable
+  - simple-scalable
+  - installation
+---
+
+<!-- Refer to [Front matter]({{< relref "../../front-matter/" >}}) for more information about how to populate front matter. -->
+
+## Tasks
+
+* Install scalable Loki with Helm
+* Install single binary Loki
+* Configure object store
+* Configure Minio instead of object store
+* Configure monitoring and alerts.
+* Install Canary
+* Configure monitoring and alerts for Canary.
+* Install Cloud integration for meta-monitoring
+
+## Concepts
+
+* Single Binary and Scalable Installation
+* object store vs Minio vs filesystem
+* Service and self monitoring, aka meta monitoring
+* Canary
```

Comment:
@osg-grafana these are just the tasks and concepts I've identified so far. We can discuss more on Monday and you can give me some details on how you imagine the docs to be like :slightly_smiling_face: